test/unit/metrics/test_fairness_metrics.py

```diff
@@ -155,6 +158,44 @@ def test_equalized_odds_ratio_weighted(agg_method, agg):
         assert actual == ratios.mean()
 
 
+@pytest.mark.parametrize(
+    ["agg", "expectation"],
+    [
+        (
+            "wrong-arg",
+            pytest.raises(
+                ValueError, match="agg must be one of 'worst_case' or 'mean', got wrong-arg"
+            ),
+        ),
+        ("mean", does_not_raise()),
+        ("worst_case", does_not_raise()),
+    ],
+)
+def test_equalized_odds_difference_raises_on_invalid_agg(
+    agg: str, expectation: AbstractContextManager
+):
+    with expectation:
+        equalized_odds_difference(y_t, y_p, sensitive_features=g_1, agg=agg)
+
+
+@pytest.mark.parametrize(
+    ["agg", "expectation"],
+    [
+        (
+            "wrong-arg",
+            pytest.raises(
+                ValueError, match="agg must be one of 'worst_case' or 'mean', got wrong-arg"
+            ),
+        ),
+        ("mean", does_not_raise()),
+        ("worst_case", does_not_raise()),
+    ],
+)
```

Comment:
A suggestion: combining these two tests into one by having the function be a parameter too: 
`@pytest.mar.parametrize('method', [equalized_odds_difference, equalized_odds_ratio])`